src/tokens/evo-light.css

```diff
@@ -196,4 +196,24 @@
         var(--color-ai-solid-green-subtle) 100%
     );
     --color-media-disabled-filter: grayscale(1) opacity(0.25);
+    /* Motion Functions */
```

Comment:
Do we need these in dark mode? Or will the light mode ones be picked up as well. 
Maybe its better that they go in the core file but not sure.

Reply:
Let's leave them as is for now. There are many questions to be ironed out internally before we decide how to organize these.